projects/ngx-openlayers/package.json

```diff
@@ -8,9 +8,8 @@
     "postchangelog": "cp projects/ngx-openlayers/CHANGELOG.md dist/ngx-openlayers/CHANGELOG.md"
   },
   "peerDependencies": {
-    "@angular/common": "^6.0.0-rc.0 || ^6.0.0 || ^7.0.0",
-    "@angular/core": "^6.0.0-rc.0 || ^6.0.0 || ^7.0.0",
-    "ol": "^5.3.1"
-  },
-  "dependencies": {}
+    "@angular/common": "^6.0.0 || ^7.0.0 || ^8.0.0",
+    "@angular/core": "^6.0.0 || ^7.0.0 || ^8.0.0",
+    "ol": "^6.0.0"
```

Comment:
Why openlayers would be a peer dependency instead of a dependency?

Reply:
This lib is a wrapper, it does not directly import openlayer.
https://yarnpkg.com/lang/en/docs/dependency-types/#toc-peerdependencies

Reply:
But in an angular project, if I want to work with ngx-openlayers, I wouldn't have to also add openlayers. Check all angular libs that wrap third party libs, these latter are always declared as dependencies, not peer.

Reply:
`@angular/fire`:
```json
"peerDependencies": {
    "@angular/common": ">=6.0.0 <9 || ^9.0.0-0",
    "@angular/core": ">=6.0.0 <9 || ^9.0.0-0",
    "@angular/platform-browser": ">=6.0.0 <9 || ^9.0.0-0",
    "@angular/platform-browser-dynamic": ">=6.0.0 <9 || ^9.0.0-0",
    "firebase": ">= 5.5.7 <7",
    "rxjs": "^6.0.0"
  },
```

Reply:
`@ngrx/effects`:
```json
"peerDependencies": {
    "@angular/core": "^8.0.0",
    "@ngrx/store": "8.4.0",
    "rxjs": "^6.4.0"
  },
```

Reply:
`@angular/material`:
```json
"peerDependencies": {
    "@angular/animations": "^8.0.0 || ^9.0.0-0",
    "@angular/cdk": "8.2.3",
    "@angular/core": "^8.0.0 || ^9.0.0-0",
    "@angular/common": "^8.0.0 || ^9.0.0-0",
    "@angular/forms": "^8.0.0 || ^9.0.0-0"
  },
```

Reply:
Ok, there's pros and cons, but now I'm more convinced by the pros :)
So as discussed, leave it as peer dependencies, we just keep in mind to update install command in README with: `npm install openlayers ngx-openlayers --save`